### PR TITLE
copypasta

### PR DESCRIPTION
**Pull request by MikeWrock**

### CODE REVIEW

File: src/moveit_pro_ur_configs/arm_on_rail_sim/objectives/wrist_snap.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<root BTCPP_format="4" main_tree_to_execute="Take wrist camera snapshot">
+<root BTCPP_format="4" main_tree_to_execute=" Take wrist camera snapshot">
```
Comment (MikeWrock) [inline src/moveit_pro_ur_configs/arm_on_rail_sim/objectives/wrist_snap.xml:2]:
```suggestion
<root BTCPP_format="4" main_tree_to_execute="Take wrist camera snapshot">
```

File: src/moveit_pro_ur_configs/picknik_ur_base_config/objectives/request_teleoperation.xml [hunk 2]
```diff
@@ -64,6 +64,10 @@
             <!--Interactive markers move to pose-->
             <Decorator ID="ForceSuccess" _while="teleop_mode == 4">
               <Control ID="Sequence">
+                <Action
+                  ID="SwitchUIPrimaryView"
+                  primary_view_name="Visualization"
+                />
```
Comment (MikeWrock) [inline src/moveit_pro_ur_configs/picknik_ur_base_config/objectives/request_teleoperation.xml:70]:
```suggestion
```

File: src/moveit_pro_ur_configs/picknik_ur_base_config/objectives/request_teleoperation.xml [hunk 3]
```diff
@@ -98,6 +102,10 @@
             <!--Waypoint buttons move to joint state-->
             <Decorator ID="ForceSuccess" _while="teleop_mode == 3">
               <Control ID="Sequence">
+                <Action
+                  ID="SwitchUIPrimaryView"
+                  primary_view_name="Visualization"
+                />
```
Comment (MikeWrock) [inline src/moveit_pro_ur_configs/picknik_ur_base_config/objectives/request_teleoperation.xml:108]:
```suggestion
```

File: src/moveit_pro_ur_configs/picknik_ur_base_config/objectives/request_teleoperation.xml [hunk 4]
```diff
@@ -131,6 +139,10 @@
             </Decorator>
             <!--Cartesian and joint jogging-->
             <Control ID="Sequence" _while="teleop_mode == 2">
+              <Action
+                ID="SwitchUIPrimaryView"
+                primary_view_name="Visualization"
+              />
```
Comment (MikeWrock) [inline src/moveit_pro_ur_configs/picknik_ur_base_config/objectives/request_teleoperation.xml:145]:
```suggestion
```

File: src/moveit_pro_ur_configs/picknik_ur_base_config/objectives/reset_planning_scene.xml [hunk 5]
```diff
@@ -3,10 +3,9 @@
   <!-- ////////// -->
   <BehaviorTree
     ID="Reset Planning Scene"
-    _description="Remove all runtime additions to the planning scene"
+    _description=""
     _favorite="false"
     _hardcoded="false"
```
Comment (MikeWrock) [inline src/moveit_pro_ur_configs/picknik_ur_base_config/objectives/reset_planning_scene.xml:8]:
```suggestion
    _description="Remove all runtime additions to the planning scene"
    _favorite="false"
    _hardcoded="false"
    _subtreeOnly="false"
```

**Comment by MikeWrock** [comment]
Closes https://github.com/PickNikRobotics/moveit_studio/issues/9581

**Review by davetcoleman — APPROVED** [review]
Skimmed, looks like the prev PR I approved. I did not test.

**Comment by mewtwoshaurd** [comment]
just checking that we got all of them? There are some in lab_sim, like `3_waypoint_pick_and_place` that we have categorized in the `arm_on_rail_sim` but not in the `lab_sim` -- do we plan on just inheriting those or something later?

**Comment by MikeWrock** [comment]
@mewtwoshaurd I must have missed the ones that didnt have subtree tags in them

**Pull request merged**